docs/running_a_warnet_scenario.md

```diff
@@ -0,0 +1,91 @@
+# Running a warnet scenario
+
+Warnet allows a bitcoin network topology and scenario steps to be defined in code. This document will use an existing scenario that demonstrates an inventory denial-of-service attack ([Inv DOS](https://invdos.net/)) on Bitcoin Core 16.1. This particular vulnerability was patched in 16.2 and later. Warnet scenarios can be any conceivable network toplogy and series of steps; they aren't just useful for demonstrating vulnerabilities but could be used to demonstrate new functionality instead.
+
+## Clone example scenario
+
+```shell
+git clone https://github.com/pinheadmz/invdos-warnet
```

Comment:
Perhaps choose another example scenario here

Reply:
OK, when https://github.com/bitcoin-dev-project/warnet/issues/559 is written I will switch the doc to be that scenario